pkg/netns/netns_linux.go

```diff
@@ -126,35 +126,39 @@ func NewNS() (ns.NetNS, error) {
 		// Don't unlock. By not unlocking, golang will kill the OS thread when the
 		// goroutine is done (for go1.10+)
 
+		threadNsPath := getCurrentThreadNetNSPath()
+
 		var origNS ns.NetNS
-		origNS, err = ns.GetNS(getCurrentThreadNetNSPath())
+		origNS, err = ns.GetNS(threadNsPath)
 		if err != nil {
+			logrus.Warnf("cannot open current network namespace %s: %q", threadNsPath, err)
 			return
 		}
-		defer func() {
-			if err := origNS.Close(); err != nil {
```

Comment:
Why are we dropping the close here?

Reply:
my mistake.  Thanks for spotting it.  Pushed a fixed version

Reply:
I don't see the fix?